site/content/extend/plugins/developer-workflow.md

```diff
@@ -16,6 +16,7 @@ subsection: Plugins
 - `make disable` - Disables the plugin on the Mattermost server
 - `make reset` - Disables and re-enables the plugin on the Mattermost server
 - `make debug-plugin` - Starts a `delve` process and attaches it to your running plugin
+- `make clean` - Force deletes recursively the contents of `dist`, `server/coverage.txt`, `server/dist`, `webapp/junit.xml`, `webapp/dist`, `webapp/node_modules` and `build/bin/` folders
```

Comment:
I'm a bit hesitant to name out the directories here as that information can become outdated quite easily. Maybe something like 
```suggestion
- `make clean` - Force deletes the content of build related files. Use when running into build issues,
```